### ISSUE TITLE
AddAzureJwtBearer - reference not found

### ISSUE THREAD

**Issue by v2kiran** [whoiskevinrich/SampleDotnet2.0GlobalFilterProject#1]
Hi thank you for this sample. I am trying to hook this up but getting an error about the reference. Was this supposed to be another extension method to "Microsoft.AspNetCore.Authentication"?

also as per this comment:
// bind the "AzureJwtBearer" Section in appsettings.json to my JwtBearerOptions

seems there should be a section in the appsettings.json called "jwtbearerOptions" that is missing. Please let me know what options need to be there.

**Comment by whoiskevinrich** [comment]
Right you were. I tried to do it both ways. I had originally constructed a configuration extension to behave much like the `.AddAzureAd()` extension provided by Microsoft, but figured it was outside the scope of what I was trying to demonstrate. I've removed the offending code, which did what the `.AddJwtBearer(options ...)` extension did anyway.

Let me know if you have further issues.

Thanks!
